backend/src/test/java/com/empire/Mocks.java

```diff
@@ -22,18 +22,4 @@ public static Noble noble(String nobleTag, double unrest) {
         when(n.hasTag(nobleTag)).thenReturn(true);
         return n;
     }
-
-    public static Character character(double general, double admiral, double governor, double spy) {
```

Comment:
Curious, why are you taking the mocks out of the unit tests?  This breaks the "unit" idea of unit testing.

Reply:
Since experience -> effect is now an implementation detail of Character, I didn't like these mocks. However, you're right that I don't need to use the real Character classes - please take another look.